### PR TITLE
[CARBG-43] Modify npm readme with t d documentation

### PR DESCRIPTION
**Pull request by NicoMeleThinkAndDev**
- The readme has been modified for uploading to the new version of the NPM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,27 @@
...
+Is a groundbreaking tool that is transforming digital board game development. It smartly combines traditional game design with modern blockchain technology. This framework is designed for developers who are familiar with boardgame.io but want to improve their projects with better security and decentralization.
+At its core, T&D’s Boardgame Framework builds on boardgame.io, a popular library known for its versatility in creating various board games. What makes T&D’s Boardgame Framework unique is its integration with blockchain technology, specifically using Cartesi's capabilities.
 
-### Features
+The main feature of this framework is its ability to validate game moves using blockchain. This is achieved through two key Cartesi technologies: the Cartesi Virtual Machine (CVM) and Cartesify. 
+The CVM is an advanced system that allows complex computations to happen off-chain while keeping blockchain-level security. Cartesify, on the other hand, is a tool that simplifies the process of adapting existing applications to run in the Cartesi environment.
+
+
+
+### WiP (Work in Progress)
```
Comment (jm-aragonpaz) [inline README.md:25]: It should clarify what is already implemented and what is on the roadmap. Maybe some sections should not be listed here.

File: README.md [hunk 2]
```diff
@@ -48,61 +38,15 @@ networking or storage code.
...
-npm install boardgame.io
+npm install @think-and-dev/cartesi-boardgame
 ```
 
-### Documentation
+### Documentation's boardgame.io
 
 Read our [Full Documentation](https://boardgame.io/documentation/) to learn how to
 use boardgame.io, and join the [community on gitter](https://gitter.im/boardgame-io/General)
 to ask your questions!
```
Comment (jm-aragonpaz) [inline README.md:47]: Maybe a section or a comment stating the Cartesi Grant Program support would be a nice addition.

Comment (jm-aragonpaz) [inline README.md:47]: Also, a comment that makes it clear that this is an early beta version and does not have all of the features yet might be a good addition.

**Review by jm-aragonpaz — COMMENTED** [review]
Great job! Left some comments

**Review by lucas-marc — APPROVED** [review]

**Pull request merged**